addons/gamification/security/ir.model.access.csv

```diff
@@ -2,18 +2,26 @@ id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
 
 goal_employee,"Goal Employee",model_gamification_goal,base.group_user,1,1,0,0
 goal_manager,"Goal Manager",model_gamification_goal,base.group_erp_manager,1,1,1,1
+goal_portal,"Goal Portal",gamification.model_gamification_goal,base.group_portal,1,1,0,0
```

Comment:
I don't think we have clear spec about how to write security rule names. I use something like views: gamification.goal.group.portal, but I let you decide to burn in hell or rise like a phoenix.